spec/factories/users.rb

```diff
@@ -0,0 +1,8 @@
+FactoryBot.define do
+  factory :user do
+    firstname { "Denis" }
+    lastname { "Zaharov" }
+    email { "denis.zaharov@flatstack.com" }
```

Comment:
I think you can use faker here because email should be uniq. If you try to create more than 1 users in one test you will receive validation error.
```suggestion
    email { Faker::Internet.email }
```